dicebot/tasks/unban.py

```diff
@@ -50,7 +50,7 @@ async def unban_async(
             if ban_id is not None and current_ban.id != ban_id:
                 return
 
-            await channel.send(f"<@{target_id}>: You have been unbanned.")
+            await channel.send(f"<@{target_id}>: You have been unbanned.", silent=True)
```

Comment:
Isn't `silent` already the default here?

Reply:
I don't believe so. It's only the default when using `ctx.send`, and this just sends to `channel` directly.

Reply:
Oh my b, I didn't realize this was `channel.send` instead of `ctx.send`. Yeah, this is in the async task handler code.